.github/retag-and-push.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```

Comment:
You might be able to find a github actions flow ready made to do this. 
I found one for pushing to docker hub on a tag the other day! 

https://docs.github.com/en/actions/publishing-packages/publishing-docker-images

Reply:
Gonna stick with this method for now - its the semver that is the tricky bit anyhow :+1: